watchmaker/managers/base.py

```diff
@@ -191,40 +211,49 @@ def download_file(self, url, filename, sourceiss3bucket=False):
             except Exception as exc:
                 # TODO: Update `except` logic
                 logging.error('Unable to download file from web server.\n'
-                                  'url = {0}\n'
-                                  'filename = {1}\n'
-                                  'Exception: {2}'
-                                  .format(url, filename, exc))
+                              'url = {0}\n'
+                              'filename = {1}\n'
+                              'Exception: {2}'
+                              .format(url, filename, exc))
                 raise SystemError('Unable to download file from web server.\n'
                                   'url = {0}\n'
                                   'filename = {1}\n'
                                   'Exception: {2}'
                                   .format(url, filename, exc))
             logging.debug('Downloaded file from web server -- \n'
-                  '    url      = {0}\n'
-                  '    filename = {1}'.format(url, filename))
+                          '    url      = {0}\n'
+                          '    filename = {1}'.format(url, filename))
+
+    def call_process(self, cmd):
+        if not isinstance(cmd, list):
+            exceptionhandler('Command is not a list.\n{0}'.format(str(cmd)))
+        rsp = subprocess.call(cmd)
+
+        if rsp != 0:
```

Comment:
`def call_process(self, cmd):
     if not isinstance(cmd, list):
          exceptionhandler('Command is not a list.\n{0}'.format(str(cmd)))
     process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     out, err = process.communicate()
     if process.returncode != 0:
          exceptionhandler('Command execution failed.\n{0}\nError Message:{1}'.format(cmd, err)
     else:
          logging.info(out)`

I wrote this in the browser... so if there is any off syntax, ill catch it before commit.  This changes the flow a little.  It will either dump if the command execution failed and carry the command used and return the shell error in the log and output in the exceptionhandler, or it will log any output that may have come from the command.